packages/extension/src/pages/sign/components/messages/registry.tsx

```diff
@@ -94,6 +95,7 @@ const UnknownMessageContent: FunctionComponent = ({ children }) => {
 };
 
 export const defaultRegistry = new MessageRenderRegistry();
+defaultRegistry.register(AgoricWalletSpendActionMessage);
```

Comment:
I think you forgot to add agoric related `.proto` files?

Reply:
You're right, I've now included the relevant `.proto` file, copied from https://github.com/Agoric/agoric-sdk/blob/master/golang/cosmos/proto/agoric/swingset/msgs.proto